notes.md

```diff
@@ -1,13 +1,18 @@
 T&P
 
-users should be able to sign up and log in
+To run server
 
-users should be able to add their tea and pronoun preferences
+```sh
+rails s
+```
+To start and run postgres database via docker:
 
-users should be able to view other user's tea and pronoun preferences
+```sh
+docker run postgres
```

Comment:
This shouldn't be in the repo, it's entirely specific to the user's machine. Each dev will do this differently.